xbmc/utils/StringUtils.cpp

```diff
@@ -1072,49 +1076,8 @@ void StringUtils::WordToDigits(std::string &word)
 
 std::string StringUtils::CreateUUID()
 {
-  /* This function generate a DCE 1.1, ISO/IEC 11578:1996 and IETF RFC-4122
-  * Version 4 conform local unique UUID based upon random number generation.
-  */
-  char UuidStrTmp[40];
-  char *pUuidStr = UuidStrTmp;
-  int i;
-
-  static bool m_uuidInitialized = false;
-  if (!m_uuidInitialized)
-  {
-    /* use current time as the seed for rand()*/
-    srand(time(NULL));
-    m_uuidInitialized = true;
-  }
-
-  /*Data1 - 8 characters.*/
-  for(i = 0; i < 8; i++, pUuidStr++)
-    ((*pUuidStr = (rand() % 16)) < 10) ? *pUuidStr += 48 : *pUuidStr += 55;
-
-  /*Data2 - 4 characters.*/
-  *pUuidStr++ = '-';
-  for(i = 0; i < 4; i++, pUuidStr++)
-    ((*pUuidStr = (rand() % 16)) < 10) ? *pUuidStr += 48 : *pUuidStr += 55;
-
-  /*Data3 - 4 characters.*/
-  *pUuidStr++ = '-';
-  for(i = 0; i < 4; i++, pUuidStr++)
-    ((*pUuidStr = (rand() % 16)) < 10) ? *pUuidStr += 48 : *pUuidStr += 55;
-
-  /*Data4 - 4 characters.*/
-  *pUuidStr++ = '-';
-  for(i = 0; i < 4; i++, pUuidStr++)
-    ((*pUuidStr = (rand() % 16)) < 10) ? *pUuidStr += 48 : *pUuidStr += 55;
-
-  /*Data5 - 12 characters.*/
-  *pUuidStr++ = '-';
-  for(i = 0; i < 12; i++, pUuidStr++)
-    ((*pUuidStr = (rand() % 16)) < 10) ? *pUuidStr += 48 : *pUuidStr += 55;
-
-  *pUuidStr = '\0';
-
-  m_lastUUID = UuidStrTmp;
-  return UuidStrTmp;
+  boost::uuids::uuid uuid = boost::uuids::random_generator()();
```

Comment:
you could use auto here to cut down on the uuids a bit, matter of taste though.
maybe add a comment that yes ()() is actually correct.